frontend/client-list/toolbar/clients-table-toolbar.component.tsx

```diff
@@ -122,4 +230,12 @@ type ClientsTableToolbarProps = {
   setPage(pageNum: number): void;
   setSearch(searchParse: SearchParseValues): any;
   fetchingClient: boolean;
+  selectedClients: any;
```

Comment:
I wasn't sure if this should be Array<object> or not

Reply:
yeah `Array<ClientListClient>`